utils/eventtracking.js

```diff
@@ -109,6 +109,53 @@ const getEventTrackingConsent = async () => {
     }
 };
 
+const getIdTrackingConsent = async () => {
+    const rl = readline.createInterface({
+        input: process.stdin,
+        output: process.stdout,
+    });
+    try {
+        for (let attempts = 0; attempts < 10; attempts++) {
+            const answer = await new Promise((resolve) => {
+                rl.question(
+                    chalk`We would like to help with your development journey with NEAR.` +
+                        chalk` We will ask you to expose your account Id while using command. ` +
```

Comment:
Do you think we could replace "expose" with "share" here and the next line?

Reply:
yes, definitely, good idea, thanks